### ISSUE TITLE
[Mod] names output parses links/invites

### ISSUE THREAD

**Issue by MeatyChunks** [Cog-Creators/Red-DiscordBot#2555]
# Command bugs

#### Command name

names

#### What cog is this command from?

Mod

#### What were you expecting to happen?

As a precaution Discord invites/URLs should be cleaned on the command's output.

#### What actually happened?

If a user sets their name to a discord invite or other URL it's parsed on the list of past nicknames.

#### How can we reproduce this issue?

Set a user's nickname to a url/invite, then use the names command on them.

**Comment by mikeshardmind** [comment]
`[p]userinfo` Has prior nickname builtin with invites filtered. I think having access to the unfiltered names for mod use in `[p]names` is still useful. I suggest locking the command to your mod/admin role using permissions.

**Comment by Dav-Git** [comment]
What are our thoughts on this? Personally agree with Sinbad here. Should this be closed or shall I make a PR?

**Comment by Jackenmen** [comment]
I agree that having access to unfiltered names is indeed useful. We could, however, perhaps lock the command to the mod role by default. I'm not entirely sure it is a good idea and would appreciate more opinions here.
